### PR TITLE
Let Travis publish bower-chosen

### PR DESCRIPTION
**Pull request by pfiller**
@harvesthq/chosen-developers 

On master commits, bower-publish.sh will push all changes in compiled files straight to harvesthq/bower-chosen. If Chosen's version has changed, the repo will also get tagged with the latest version.

Thanks for the :fire: tip, @stof!

### CODE REVIEW

File: Gruntfile.coffee [hunk 1]
```diff
@@ -90,20 +90,13 @@ This file is generated by `grunt build`, do not edit it by hand.
       options:
         base: 'public',
         message: "Updated to new Chosen version #{version()}"
-      'gh-pages':
-        src: ['**']
-      'bower':
```
Comment (pfiller) [inline Gruntfile.coffee]: We opted to use a script instead of gh-pages as it was easier to avoid tagging on every push.

File: Gruntfile.coffee [hunk 2]
```diff
@@ -90,20 +90,13 @@ This file is generated by `grunt build`, do not edit it by hand.
...
-        options:
-          repo: 'https://github.com/harvesthq/bower-chosen'
-          tag: version_tag()
-        src: ['bower.json', 'chosen.jquery.min.js', 'chosen.min.css', 'chosen-sprite.png', 'chosen-sprite@2x.png']
+      src: ['**']
 
   grunt.registerTask 'default', ['build']
-  grunt.registerTask 'build', ['coffee', 'compass', 'concat', 'uglify', 'cssmin']
+  grunt.registerTask 'build', ['coffee', 'compass', 'concat', 'uglify', 'cssmin', 'package-bower']
```
Comment (pfiller) [inline Gruntfile.coffee]: All `package-bower` does is build the bower.json file. It's harmless!

File: bower-publish.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+CURRENT_BRANCH=`git rev-parse --abbrev-ref HEAD`
+
+if [ $CURRENT_BRANCH != 'master' ] ; then
+  exit 0
```
Comment (adunkman) [inline bower-publish.sh]: It’d be nice if the script always output something — maybe “This build isn’t on master. Skipping bower-chosen release.” here?

**Comment by adunkman** [comment]
:heart: :heart: :heart: 

+1!

**Pull request merged**